apps/cowswap-frontend/src/modules/tradeFormValidation/pure/TradeFormButtons/tradeButtonsMap.tsx

```diff
@@ -105,8 +105,22 @@ export const tradeButtonsMap: Record<TradeFormValidation, ButtonErrorConfig | Bu
   [TradeFormValidation.BrowserOffline]: {
     text: 'Error loading price. You are currently offline.',
   },
-  [TradeFormValidation.RecipientInvalid]: {
-    text: 'Enter a valid recipient',
+  [TradeFormValidation.RecipientInvalid]: ({ derivedState: { inputCurrency, outputCurrency, recipient } }) => {
+    const isBridging = inputCurrency && outputCurrency && inputCurrency.chainId !== outputCurrency.chainId
+
+    return (
+      <TradeFormBlankButton disabled>
+        <>
+          <Trans>Enter a valid recipient</Trans>
+          {isBridging && recipient && (
+            <HelpTooltip
+              placement="top"
+              text="ENS recipient not supported for Swap and Bridge — use address instead."
```

Comment:
Cross-chain swaps?
We might need a review of all text related to it as that's what marketing is calling it.